CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # osquery Changelog
 
+
+<a name="5.2.1"></a>
+## [5.2.1](https://github.com/osquery/osquery/releases/tag/5.2.1)
+
+[Git Commits](https://github.com/osquery/osquery/compare/5.1.0...5.2.1)
+
+Osquery 5.2.1 brings native m1 support to the macOS platform. 
+
+Representing commits from 9 contributors! Thank you all.
+
+### New Features
+
+- Apple Silicon support ([#7330](https://github.com/osquery/osquery/pull/7330))
+
+### Table Changes
+
+- Update `time` table to reflect UTC values ([#7276](https://github.com/osquery/osquery/pull/7276))
+- Hide the deprecate `antispyware` column in `windows_security_center` ([#7411](https://github.com/osquery/osquery/pull/7411))
+
+### Under the Hood improvements
+
+
+### Bug Fixes
+
+
+- Update the ATC table `path` column check to be case insensitive ([#7442](https://github.com/osquery/osquery/pull/7442))
+- Fix a crash when Yara uses its own strutils functions ([#7439](https://github.com/osquery/osquery/pull/7439))
```

Comment:
If #7439 fixes a regression added in 5.2.0, but 5.2.0 was not released, does this still go in the changelog? Your call I guess

Reply:
My general rubric is that all commits go into the changelog. But I should reword this one to be clear.